### PR TITLE
fix: formatter breaks php-view file contains phpdoc block and php short-tags

### PR DESCRIPTION
**Pull request by zarv1k**
Fixes: #1112
Reviewers: @evilebottnawi

### CODE REVIEW

**Comment by zarv1k** [comment]
@evilebottnawi could you please review the PR?

File: tests/inline/__snapshots__/jsfmt.spec.js.snap [hunk 1]
```diff
@@ -2347,6 +2347,58 @@ try {
...
+<?php
+/**
+ * @var string $a
+ */
+?>
+<?php echo $a; ?>
+<?= $a ?>
+
+=====================================output=====================================
+<?php
```
Comment (evilebottnawi) [inline tests/inline/__snapshots__/jsfmt.spec.js.snap:2391]: Can we add test(s) where we start it with `<?=`?

Comment (zarv1k) [reply tests/inline/__snapshots__/jsfmt.spec.js.snap:2391]: added `mixed-56.php`

**Review by evilebottnawi — APPROVED** [review]

**Comment by czosel** [comment]
Looks good, thanks for contributing @zarv1k!

**Comment by czosel** [comment]
Released as v0.15.1 :tada:

**Pull request merged**